ports/squeek/port.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "squeek.zip",
+  "items": [
+    "Squeek the Meek.sh",
+    "squeek/"
```

Comment:
Why is there a "/" after "squeek"?

Reply:
Been found it this way some time now. It shows it filled out in the example

Reply:
You're right, it does show that in the example. Never seen it :)

Reply:
Ahh..ancient gamemaker secret😉😂❤